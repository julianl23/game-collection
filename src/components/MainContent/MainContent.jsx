import React from "react";
import styled from "styled-components";

const MainContent = () => {
  const Main = styled.section`
    display: grid;
    grid-template-columns: repeat(4);
    grid-gap: 16px;
    grid-auto-rows: minmax(100px, auto);
    min-height: 90vh;
    padding: 25px;
  `;

  const TempContentDiv = styled.section`
    grid-column: 1 / 4;
    grid-row: 1;
  `;

  return (
    <Main>
      <TempContentDiv>Whoa! Main Content</TempContentDiv>
    </Main>
  );
};

export default MainContent;
