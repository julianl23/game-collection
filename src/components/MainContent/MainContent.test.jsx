import React from "react";
import TestRenderer from "react-test-renderer";
import MainContent from "./MainContent";

describe("MainContent", () => {
  it("renders correctly", () => {
    const instance = TestRenderer.create(<MainContent />);
    expect(instance.toJSON()).toMatchSnapshot();
  });
});
