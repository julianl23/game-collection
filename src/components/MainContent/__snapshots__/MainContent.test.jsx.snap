// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContent renders correctly 1`] = `
.c0 {
  display: grid;
  grid-template-columns: repeat(4);
  grid-gap: 16px;
  grid-auto-rows: minmax(100px,auto);
  min-height: 90vh;
}

.c1 {
  grid-column: 1 / 4;
  grid-row: 1;
}

<section
  className="c0"
>
  <section
    className="c1"
  >
    Whoa! Main Content
  </section>
</section>
`;
